elements/pfelement/src/pfelement.js

```diff
@@ -322,19 +322,8 @@ class PFElement extends HTMLElement {
     if (window.ShadyCSS) window.ShadyCSS.styleElement(this);
 
     // If the slot definition exists, set up an observer
-    if (typeof this.slots === "object") {
+    if (typeof this.slots === "object")
```

Comment:
@kylebuch8 Can you sanity check me on this?  I was debugging looping issues in IE11 and I believe it makes sense to move the slot observer into the rendering method.  What do you think?